src/main/java/com/airlines/dao/impl/AirplaneDaoImpl.java

```diff
@@ -78,60 +81,95 @@ public AirplaneDaoImpl(DataSource dataSource) {
                         .build());
             }
         } catch (SQLException e) {
-            throw new DaoOperationException("Couldn't save product ", e);
+            throw new DaoOperationException("Couldn't find an airplane ", e);
```

Comment:
Please review all error message from AirplaneDaoImpl, then compare to CrewMemberDaoImpl and try to use some general templates.